R/froll.R

```diff
@@ -1,11 +1,42 @@
-froll = function(fun, x, n, fill=NA, algo=c("fast", "exact"), align=c("right", "left", "center"), na.rm=FALSE, hasNA=NA, adaptive=FALSE) {
+partial2adaptive = function(x, n, align) {
```

Comment:
I found myself writing out some example input/output by hand, esp. for the `is.list(n)` case, maybe write some in a comment to simplify for future readers can help

Reply:
added examples for supported usage